File: standalone_app_v3_6_go/internal/planner/planner.go
package planner

import (
	"image"

	"eyesam/standalone_app_v3_6_go/internal/utils"
)

func ComputeFAZCenter(mask utils.Mask) image.Point {
	fazBin := utils.Binarize(mask, 1)
	fazLcc := utils.LargestConnectedComponent(fazBin)
	return utils.InscribedCenter(fazLcc)
}

File: standalone_app_v3_7/planner.py
from typing import Tuple

import numpy as np

from .utils import binarize_mask, inscribed_center, largest_connected_component


def compute_faz_center(faz_mask: np.ndarray) -> Tuple[int, int]:
    faz_bin = binarize_mask(faz_mask)
    faz_lcc = largest_connected_component(faz_bin)
    return inscribed_center(faz_lcc)
